packages/shell/src/index.ts

```diff
@@ -32,15 +31,13 @@ await app.initializeKeys();
   const location = new URL(globalThis.location.href);
   const segments = location.pathname.split("/");
   segments.shift(); // shift off the pathnames' prefix "/";
-  let [spaceName, charmId] = USE_SHELL_PREFIX
-    ? [segments[1], segments[2]]
-    : [segments[0], segments[1]];
+  let [spaceName, charmId] = [segments[0], segments[1]];
   if (!spaceName) {
     spaceName = "common-knowledge";
     globalThis.history.replaceState(
       {},
       "",
-      USE_SHELL_PREFIX ? "/shell/common-knowledge" : "/common-knowledge",
+      "/common-knowledge",
     );
   }
```

Comment:
Needs rebased from the `Navigation` work; edit: oh this should've been removed with the `Navigation` work -- I'll look into it